app/controllers/publishers_controller.rb

```diff
@@ -423,6 +423,43 @@ def balance
     end
   end
 
+  def banner_editor_data
+    # Handle edge case where user hasn't created channels yet.
+    no_channels_banner = current_publisher.site_banners.find_by(channel_id: "00000000-0000-0000-0000-000000000000")
+    if no_channels_banner && !current_publisher.channels.length.zero?
+      no_channels_banner.delete
+    end
+
+    channels = []
+    current_publisher.channels.each do |c|
+      channel = {
+                "id" => c.id,
+                "name" => c.publication_title,
+                "type" => c.details_type
+                }
+      channels.push(channel)
+    end
+    # Handle edge case where user hasn't created channels yet.
+    if current_publisher.channels.length.zero?
+      data = no_channels_banner_data
+      return render(json: data.to_json)
```

Comment:
`render(json: data.to_json) and return` is the convention

Reply:
Really? I know it's a convention throughout the project but I was thinking that Rubocop/the community didn't like it.

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/AndOr

Reply:
Done ✔️